confgenerator/logging_receivers.go

```diff
@@ -228,3 +228,28 @@ func (r LoggingReceiverWindowsEventLog) Components(tag string) []fluentbit.Compo
 func init() {
 	LoggingReceiverTypes.RegisterType(func() Component { return &LoggingReceiverWindowsEventLog{} }, "windows")
 }
+
+// A LoggingReceiverSystemd represents the user configuration for a Systemd/journald receiver.
+type LoggingReceiverSystemd struct {
```

Comment:
Why is there no configuration for this receiver? Without the ability to configure journald matches it's going to collect a bunch of redundant log messages.

Reply:
Detailed reasoning for the decision was recorded in the internal design doc `go/ops-agent-systemd-receiver-design`.